apps/site/assets/ts/stop/components/DepartureCard.tsx

```diff
@@ -34,22 +34,28 @@ const DepartureCard = ({
         {routeName(route)}
       </a>
       {/* TODO can we avoid hard coding the direction ids? */}
-      <DepartureTimes
-        key={`${route.id}-0`}
-        route={route}
-        directionId={0}
-        departuresForDirection={departuresByDirection[0]}
-        onClick={onClick}
-        alertsForDirection={allAlertsForDirection(alertsForRoute, 0)}
-      />
-      <DepartureTimes
-        key={`${route.id}-1`}
-        route={route}
-        directionId={1}
-        departuresForDirection={departuresByDirection[1]}
-        onClick={onClick}
-        alertsForDirection={allAlertsForDirection(alertsForRoute, 1)}
-      />
+      {!departuresByDirection[0] && !departuresByDirection[1] ? (
```

Comment:
I think it'd be better to treat each direction independently, so that if (for example) direction `1` has departures and `0` doesn't, we don't hide the departure times for `1`

Reply:
@thecristen do you have a link to the designs for that screenshot?
I think this is what's currently happening. The `No upcoming trips` only shows if both directions have no upcoming trips. If either has trips it goes to render departure times